habushu-maven-plugin/src/test/resources/specifications/poetry-core-version-migration.feature

```diff
@@ -2,22 +2,32 @@ Feature: Poetry-core version migration
   Baton will migrate the poetry-core version within the pyproject.toml file to match the required version needed by the Habushu.
 
 
-Scenario: Migrate poetry-core version when existing poetry-core version is less than 1.6.0
-Given an existing pyproject.toml file with poetry-core version less than 1.6.0 in the build-system group
-When Habushu poetry core migration executes
-Then the poetry-core version is updated to 1.6.0 in the build-system group
+  Scenario: Migrate poetry-core version when existing poetry-core version is less than 1.6.0
+    Given an existing pyproject.toml file with poetry-core version less than 1.6.0 in the build-system group
+    When Habushu poetry core migration executes
+    Then the poetry-core version is updated to 1.6.0 in the build-system group
 
-Scenario: No migration will be performed when the poetry-core version is equal to 1.6.0
-Given an existing pyproject.toml file with poetry-core version of 1.6.0 in the build-system group
-When Habushu poetry core migration executes
-Then no update was performed for poetry-core version
+  Scenario: No migration will be performed when the poetry-core version is equal to 1.6.0
+    Given an existing pyproject.toml file with poetry-core version of 1.6.0 in the build-system group
+    When Habushu poetry core migration executes
+    Then no update was performed for poetry-core version
 
-Scenario: No migration will be performed when the poetry-core version of 1.7.0
-Given an existing pyproject.toml file with poetry-core version of 1.7.0 in the build-system group
-When Habushu poetry core migration executes
-Then no update was performed for poetry-core version
+  Scenario: No migration will be performed when the poetry-core version of 1.7.0
+    Given an existing pyproject.toml file with poetry-core version of 1.7.0 in the build-system group
+    When Habushu poetry core migration executes
+    Then no update was performed for poetry-core version
 
-Scenario: No migration will be performed when the poetry-core version of 2.0.0
-Given an existing pyproject.toml file with poetry-core version of 2.0.0 in the build-system group
-When Habushu poetry core migration executes
-Then no update was performed for poetry-core version
\ No newline at end of file
+  Scenario: No migration will be performed when the poetry-core version of 2.0.0
+    Given an existing pyproject.toml file with poetry-core version of 2.0.0 in the build-system group
+    When Habushu poetry core migration executes
```

Comment:
I: recommend ignoring whitespace

Reply:
I don't understand this comment. Could you elaborate a little further? Will try to see if I can fix it.

Reply:
Not affecting any functionality, so merging it. will fix in other tickets if this becomes an issue.

Reply:
I - Informational.  This is for someone else reading this.  80% of the changes in the file where whitespace, so I'm just trying to help the next person out interpreting it.

Reply:
Got it. Thanks.